### PR TITLE
Jira Issue Collector behavior

### PR DESCRIPTION
**Pull request by besseym**
function to add Jira Issue Collector behavior

### CODE REVIEW

**Comment by kanastasakis26** [comment]
I am little confused as to the flow. 
An Ajax request is made, but there is no clear handler. 
And I cannot see how window.ATL_JQ_PAGE_PROPS comes into play

**Comment by besseym** [comment]
> I am little confused as to the flow.
> An Ajax request is made, but there is no clear handler.
> And I cannot see how window.ATL_JQ_PAGE_PROPS comes into play

I'm not sure how it works either. This is the sample code that Jira provides to add an issue collector to your site. I suppose their code, which is pulled in with the Ajax request, is looking for the ATL_JQ_PAGE_PROPS variable on the window object.

**Comment by kanastasakis26** [comment]
> > I am little confused as to the flow.
> > An Ajax request is made, but there is no clear handler.
> > And I cannot see how window.ATL_JQ_PAGE_PROPS comes into play
> 
> I'm not sure how it works either. This is the sample code that Jira provides to add an issue collector to your site. I suppose their code, which is pulled in with the Ajax request, is looking for the ATL_JQ_PAGE_PROPS variable on the window object.

Okay. Now...
If JS code is being downloaded from atlasian and then executed, will there be cross site scripting or other security problems?

File: src/ts/mmui-jira/index.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    //for Jira Issue Collector integration
+    $.ajax({
+        url: `https://massmutual.atlassian.net/s/d41d8cd98f00b204e9800998ecf8427e-T/hd2sgk/b/18/e73395c53c3b10fde2303f4bf74ffbf6/_/download/batch/com.atlassian.jira.collector.plugin.jira-issue-collector-plugin:issuecollector-embededjs/com.atlassian.jira.collector.plugin.jira-issue-collector-plugin:issuecollector-embededjs.js?locale=en-US&collectorId=${collectorId}`,
+        type: "get",
+        cache: true,
+        dataType: "script"
+    });
+
+    window.ATL_JQ_PAGE_PROPS =  {
+        "triggerFunction": function(showCollectorDialog) {
```
Comment (xyz8983) [inline src/ts/mmui-jira/index.ts]: what is showCollectorDialog?

Comment (besseym) [reply src/ts/mmui-jira/index.ts]: showCollectorDialog must be part of Jira's code that is downloaded with the ajax request.

**Pull request merged**